bin/oneoff/diff_email_previews

```diff
@@ -65,8 +68,11 @@ PREVIEWS = %w(
   teacher_enrollment_reminder__csf_intro_3_day
   teacher_enrollment_reminder__csp_1_10_day
   teacher_enrollment_reminder__csp_1_3_day
+  teacher_enrollment_reminder__csp_for_returning_teachers_10_day
```

Comment:
These are just entries we forgot to add in previous PRs, yeah? No changes to CSP for returning teachers workshop mailers in this PR, right?

Reply:
That's correct - I didn't realize we had this script, and just ran across it while working on this update.  I went ahead and updated the list of mailers it tests to reflect some recent additions.

Reply:
I wrote this script long ago to verify that email template changes don't have unexpected results. (For example, 1 partial template can be used in multiple emails that we are not aware of.) Not sure if it is helpful to anyone but me.

Reply:
Seems very useful, thanks Ha!